src/components/views/rooms/MemberTile.js

```diff
@@ -73,115 +59,50 @@ module.exports = React.createClass({
-
-            // if (this.props.member && this.props.member.powerLevelNorm > 0) {
-            //     var img = "img/p/p" + Math.floor(20 * this.props.member.powerLevelNorm / 100) + ".png";
-            //     power = <img src={ img } className="mx_MemberTile_power" width="44" height="44" alt=""/>;
-            // }
-
-            var power;
-            if (this.props.member) {
-                var powerLevel = this.props.member.powerLevel;
-                if (powerLevel >= 50 && powerLevel < 99) {
-                    power = <img src="img/mod.svg" className="mx_MemberTile_power" width="16" height="17" alt="Mod"/>;
-                }
-                if (powerLevel >= 99) {
-                    power = <img src="img/admin.svg" className="mx_MemberTile_power" width="16" height="17" alt="Admin"/>;
-                }
-            }
-        }
-
-        var mainClassName = "mx_MemberTile ";
-        mainClassName += presenceClass;
-        if (this.state.hover) {
-            mainClassName += " mx_MemberTile_hover";
-        }
+        var presenceState = member.user ? member.user.presence : null;
 
-        var nameEl;
-        if (this.state.hover && this.props.member) {
-            var presenceState = (member && member.user) ? member.user.presence : null;
-            var PresenceLabel = sdk.getComponent("rooms.PresenceLabel");
-            nameEl = (
-                <div className="mx_MemberTile_details">
-                    <img className="mx_MemberTile_chevron" src="img/member_chevron.png" width="8" height="12"/>
-                    <div className="mx_MemberTile_userId">{ name }</div>
-                    <PresenceLabel activeAgo={active}
-                        presenceState={presenceState} />
-                </div>
-            );
+        var av = (
+            <MemberAvatar member={member} width={36} height={36} />
+        );
+        var power;
+        var powerLevel = this.props.member.powerLevel;
+        if (powerLevel >= 50 && powerLevel < 99) {
+            power = <img src="img/mod.svg" className="mx_MemberTile_power" width="16" height="17" alt="Mod"/>;
         }
-        else {
-            nameEl = (
-                <div className="mx_MemberTile_name">
-                    { name }
-                </div>
-            );
+        if (powerLevel >= 99) {
+            power = <img src="img/admin.svg" className="mx_MemberTile_power" width="16" height="17" alt="Admin"/>;
         }
 
-        var MemberAvatar = sdk.getComponent('avatars.MemberAvatar');
-        var BaseAvatar = sdk.getComponent('avatars.BaseAvatar');
+        if (member.user) {
+            this.user_last_modified_time = member.user.getLastModifiedTime();
 
-        var av;
-        if (member) {
-            av = (
-                <MemberAvatar member={this.props.member} width={36} height={36} />
+            // FIXME: make presence data update whenever User.presence changes...
```

Comment:
The sort function being applied to the `MemberList` has not changed. It still calls `memberSort` which maps users based on their presence *state* and then their last_active if the states match.